hail/src/main/scala/is/hail/annotations/ValueVisitor.scala

```diff
@@ -19,6 +21,12 @@ trait ValueVisitor {
 
   def visitBinary(b: Array[Byte]): Unit
 
+  def visitLocus(l: Locus); Unit
```

Comment:
I'm pretty sure this should be a colon, not a semicolon? Not sure what the presence of a semicolon here is doing.

Reply:
huh, interesting that compiles!

Reply:
good catch.